### ISSUE TITLE
Auto-link extension doesn't find link end if ended by slash

### ISSUE THREAD

**Issue by ronnyroeller** [remirror/remirror#673]
### Summary

When entering a link with an ending slash (e.g. `www.example.com/`), the extension considers also the following content as part of the link. Removing the ending slash makes the link extension work.

### Steps to reproduce

1. Go to https://remirror.io/playground#o/N4IgtiBcAMA0IEMoG0C68CmLRgPYBMBXAGy0hACcMwBLCi3CgegwA8AXDAOwGcbcuTBIXa4AtMRpcA1iEysADo3ZQQAQRG4AMlOkBRDtz4CQAX3QgFKVKaA
2. Enter `www.example.com/ more content`

### Expected results

`www.example.com/` is a link. `more content` is normal text, i.e. not part of the link.

### Actual results

Complete string is part of the link

### Screenshot(s)

With ending slash (Bug):
![image](https://user-images.githubusercontent.com/9339055/92942776-32808300-f452-11ea-9150-3c31d23dd1d3.png)

Without ending slash (works):
![image](https://user-images.githubusercontent.com/9339055/92942843-47f5ad00-f452-11ea-958e-5c06a07cbf55.png)

**Comment by ifiokjr** [comment]
Yep noticed this a few hours ago while working on the reimplementation. I have the fix locally and will push up once this is ready.

**Comment by ifiokjr** [comment]
You can use this regex for now. 

```ts
import { AutoLinkExtension } from '@remirror/extension-auto-link';

new AutoLinkExtension({ urlRegex: /((http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[\da-z]+([.-][\da-z]+)*\.[a-z]{2,5}(:\d{1,5})?(\/\S*)?)/gi });
```
